python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -87,6 +87,10 @@ class ImageDifferenceConfig(pexConfig.Config):
         dtype=bool,
         default=True
     )
+    astrometer = pexConfig.ConfigurableField(
+        target = measAstrom.AstrometryTask,
+        doc = "astrometry task; used to match sources to reference objects, but not to fit a WCS",
+    )
```

Comment:
To answer Jim's question: it is SubaruAstrometryTask. I confess I haven't pulled obs_subaru in awhile. Paul: thank you!